### PR TITLE
[IMP] developer: document new parameter

### PR DESCRIPTION
**Pull request by mart-e**
Doc of odoo/odoo#129977

### CODE REVIEW

File: content/developer/reference/backend/mixins.rst [hunk 1]
```diff
@@ -88,6 +88,7 @@ to manage followers on your record:
         parent partners to the message in case of private discussion
     :param list(tuple(str,str)) attachments: list of attachment tuples in the form
         ``(name,content)``, where content is NOT base64 encoded
+    :param body_is_html: indicates `body` should be threated as HTML even if `str`
```
Comment (AntoineVDV) [inline content/developer/reference/backend/mixins.rst:91]:
```suggestion
    :param bool body_is_html: indicates whether `body` should be treated as HTML, even if `str`.
```

**Review by AntoineVDV — COMMENTED** [review]
@robodoo delegate+

**Comment by AntoineVDV** [comment]
This method could be `autodoc`umented to avoid such PRs and eventual discrepancies.

**Comment by mart-e** [comment]
robodoo r+

**Pull request closed without merging**